docs/config.rst

```diff
@@ -537,9 +537,11 @@ Complete list of settings that you can put into ``testenv*`` sections:
 
     .. versionadded:: 0.9
 
-    (DEPRECATED, will be removed in a future version) Multi-line ``name =
-    URL`` definitions of python package servers.  Dependencies can
-    specify using a specified index server through the
+    (DEPRECATED, will be removed in a future version) Use
```

Comment:
IMHO a better way to specify the index server is to do:

```
setenv =
    PIP_INDEX_URL = {env:PIP_INDEX_URL:https://magic.com/simple}
```

Reply:
I agree.

I've changed the commit so there's only a note at the top, following is the description of indexserver configuration option and last is the new recommendation.

I've changed magic.com/simple to pypi.org/simple to avoid someone getting magic.com. (example.com could also be used).